crates/codegen/language/internal_macros/src/derive/spanned.rs

```diff
@@ -159,7 +159,7 @@ fn get_spanned_type(input: Type) -> Type {
         }
 
         // External types should also be wrapped in 'Spanned<T>':
-        "bool" | "char" | "PathBuf" | "String" | "Version" => {
+        "bool" | "u8" | "char" | "PathBuf" | "String" | "Version" => {
```

Comment:
nit: I suggest maintaining alphabetical sorting.

Reply:
I thought these are sorted by belonging crate and size; I can sort this alphabetically.